pkg/multitenant/pgx.go

```diff
@@ -0,0 +1,13 @@
+package multitenant
+
+import "github.com/jackc/pgx/v5/pgxpool"
+
+type PgxConfigCustomization interface {
+	Customize(pgxConfig *pgxpool.Config)
+}
+
+type DefaultPgxConfigCustomization struct{}
+
+var _ PgxConfigCustomization = &DefaultPgxConfigCustomization{}
+
+func (d DefaultPgxConfigCustomization) Customize(_ *pgxpool.Config) {}
```

Comment:
Feels like `plugins/resources/postgres` is a better package for this no?

Reply:
Also worth adding the `PgxConfigCunstomizationFn` shorthand like for `Tenant`

Reply:
And probably for hashing as well, since we only have one Fn now, right?